string[] car_rank  # 车辆顺序
